examples/ciao/README.md

```diff
@@ -5,12 +5,18 @@ This is an example of a playbook to deploy CIAO using ansible.
 ## Prework
 
 ### Access
-Ansible requires that the user running the playbook has passwordless ssh access to the managed nodes, sudo privileges and python installed on each node.
+Ansible requires that the user running the playbook has passwordless ssh access
```

Comment:
In which case you need to change that sentence.  In it's current form it implies that passwordless ssh is required between the managed nodes as well.  It's also ambiguous as to whether passwordless sudo is required or not which I believe it is.  To be honest, my proposed change isn't any better.  How about this?

> Ansible requires that the user running the playbook has passwordless ssh access from the deployment machine to the managed nodes and passwordless sudo privileges on both the managed nodes and deployment machine.

This is clear and unabmiguous.